### DISCUSSION TITLE
Any guide to limit **systemwide** podman total resources?

### DISCUSSION THREAD

**Discussion by loynoir** [containers/podman#17173] [Q&A]
Any guide to limit **systemwide** podman total resources?

https://unix.stackexchange.com/questions/537645/how-to-limit-docker-total-resources

`/etc/systemd/system/docker_limit.slice`
```
[Unit]
Description=Slice that limits docker resources
Before=slices.target

[Slice]
CPUAccounting=true
CPUQuota=700%
#Memory Management
MemoryAccounting=true
MemoryLimit=25G
```

`/etc/docker/daemon.json`
```
{
  "cgroup-parent": "docker_limit.slice"
}
```

**Comment by rhatdan** [comment]
Well no, since each podman command is independent, you would have to limit the user running podman. (Not something you probably want to do for root.

Running containers under systemd unit files would allow you to further confine the containers within a slice.

Theoretically if you wanted to use only podman --remote you could setup the podman-service to match the docker service.

**Comment by giuseppe** [comment]
I think the suggestion makes sense.  The limit specified in the `.slice` unit apply to the parent cgroup where all the containers will be spawned into.

We only miss the possibility to override `cgroup-parent` from the `containers.conf` file

**Comment by jjhidalgar** [comment]
I'm going to upvote this. It should be simple if we can make all users use the same chroup parent

**Reply by giuseppe** [reply]
well, that's not possible, since each user hascits own subtree with systemd so they will still use that

**Reply by jjhidalgar** [reply]
Ouch! Thanks for clarifying
